projects/scion/workbench/src/lib/part/view-tab/view-tab.component.ts

```diff
@@ -198,7 +198,7 @@ export class ViewTabComponent {
 
   private addHostCssClasses(): void {
     const ngClass = inject(NgClass);
-    effect(() => ngClass.ngClass = this.view().classList.asList());
+    effect(() => ngClass.ngClass = this.view().classList.asList(), {forceRoot: true});
```

Comment:
Why is this required? Unlike a view, a view tab is not detached from change detection.